### DISCUSSION TITLE
Config error on my end? Cant find the cause for faulty values in HomeAssistant

### DISCUSSION THREAD

**Discussion by Meigl89** [jomjol/AI-on-the-edge-device#3276] [General]
### Issue

Hi everyone,

since I invested some time in googling, hopefully somebody here can help me. 

I setup the watermeter and configured it.

![image](https://github.com/user-attachments/assets/6cb24642-bf11-4daf-b2e5-293134f95f96)

I configured only two digits so far, since this is a new water meter. The "6" is for the x1 and the "0" for the x10.

The values are looking like this:

![image](https://github.com/user-attachments/assets/41a265cf-1333-494d-9071-a614904708b9)

The problem is now in HomeAssistant, that sometimes I get values like this one here, like yesterday:

![image](https://github.com/user-attachments/assets/6d392eed-14f9-4b1e-a35c-819304f31622)

At first look it seems that it recognizes as value times 1000. It was 6,XXXm3 and HomeAssistant sees it at as 6.XXX as it seems. But graph in home assistant for sensor.watermeter_value is the following:

![image](https://github.com/user-attachments/assets/a6be193f-f201-45c8-809d-b3019b5fc622)

And then it goes back to normal again. 

I dont know where to look anymore, maybe I've invested too much time and the error is super clear and I can't see it anymore.

Anybody an idea? 

Thanks in Advance!

Michael

**Comment by caco3** [comment]
Did you set https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-numberdecimalshift to +3?

Can you show the data graph so we can sit is like HA there as well?

**Reply by Meigl89** [reply]
It was set to default, 0. I've changed that now to +3. Now the overview is this:

![image](https://github.com/user-attachments/assets/2e41ac7b-2e22-46d1-96cd-c361e3a52b20)

Here is the data graph from the 29th, same day like in HASS.

![image](https://github.com/user-attachments/assets/a9dc704f-1818-4030-a111-df3ab9d5efd2)

On a side node, for today it was correct again (before I made changes)

![image](https://github.com/user-attachments/assets/19b9c069-57c8-4608-8412-b7ee883b41ba)

Thanks for your help!


---

After waiting for some time, I now see the following:

<img width="104" alt="image" src="https://github.com/user-attachments/assets/bb39cb13-fed4-4518-9bb8-8bfe528a85f8">


<img width="568" alt="image" src="https://github.com/user-attachments/assets/91e6c86d-cc0f-4d05-87a0-5ab0054654fe">

<img width="1025" alt="image" src="https://github.com/user-attachments/assets/b92ecc7d-ed40-4afc-9035-973112fd2129">
